main.js

```diff
@@ -1 +1,18 @@
-console.log('Hello World');
\ No newline at end of file
+const level = require('level');
+const db = connectToDatabase('./studentdb');
+
+function connectToDatabase(db){
+    return level(db, { valueEncoding: 'json' });
+}
+
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+
+acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
+
+function acceptStudent(id, fullName, age, address){
+    let attributes = { ID:id, Name: fullName, Age: age, Address: address };
+    db.put(id, attributes);
+    db.get(id, function(err, value){
+        console.log(value, status[0]);
+    })
+}
```

Comment:
you may want to make use of async /await like
```suggestion
async function acceptStudent(id, fullName, age, address){
    let attributes = { ID:id, Name: fullName, Age: age, Address: address };
    await db.put(id, attributes);
    return await db.get(id)
}
```

Reply:
the function is updated.